src/frontends/pytorch/src/op/conv_transposend.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_conv_transposend(NodeContext& context) {
+    auto strides = context.const_input<Strides>(3);
+    // In torch pads at beginning are same as at end
+    auto pads = context.const_input<CoordinateDiff>(4);
+    auto output_padding = context.const_input<CoordinateDiff>(5);
+    auto pad_type = ov::op::PadType::EXPLICIT;
+    auto dilations = context.const_input<Strides>(7);
+    auto groups = context.const_input<int64_t>(6);
```

Comment:
please check that groups number is always possitive

Reply:
it will never be scripted :) but of course I can add this check

Reply:
added